annual_dual_failure_probability.py

```
@@ -0,0 +1,111 @@
+
+    for rto_hours in [4, 24, 48, 168]:
+        joint_mtbf_hours, annual_probability = estimate_annual_dual_failure_probability(
+            system_mtbf=system_mtbf,
+            rto_hours=rto_hours,
+        )
+        print(f"RTO: {rto_hours} hours")
+        print(f"Joint MTBF: {joint_mtbf_hours:.0f} hours")
+        print(f"Annual probability of simultaneous failure: {annual_probability * 100:.5f}%")
+
+
+def redundant_mttf(mttf):
```

Ruff (0.12.2)

Missing return type annotation for public function `redundant_mttf`

(ANN201)